collectd-plugins.yaml

```diff
@@ -26,7 +26,7 @@
   repo: signalfx/collectd-health_checker
 
 - name: jenkins
-  version: v2.1.0
+  version: v2.2.0
```

Comment:
TODO before to merge this PR:
- merge https://github.com/signalfx/collectd-jenkins/pull/22
- tag the underlying repo to `v2.2.0`

Reply:
Done.